Sources/SwiftGRPC/Core/Channel.swift

```diff
@@ -136,18 +134,16 @@ private extension Channel {
     private let completionQueue: CompletionQueue
```

Comment:
Could we move `class ConnectivityObserver` into a dedicated file?

Reply:
Yep!